app/lib/ic_rails_huey_toby/errors.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module IcRailsHueyToby
+  module Errors
+    class Error < StandardError
+      attr_reader :original_error
+
+      def initialize(message = nil, original_error = nil)
+        super(message)
+
+        @original_error = original_error
+      end
+    end
+
+    class ClientServiceError < StandardError; end
```

Comment:
It might be better to be more specific:

```suggestion
    class SearchServiceError < StandardError; end
```